src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -441,4 +443,23 @@ public void selectPrivateMessageToUser(String userName) {
     PageObjectLogging.log("selectPrivateMessageToUser", "private message selected from dropdown",
             true);
   }
+
+  public Boolean isEmoticonVisible(String emoticon) {
+    try {
+      wait.forElementVisible(By.xpath(String.format(PATH_EMOTICON_ON_CHAT, emoticon)));
+      return true;
+    } catch (TimeoutException | NoSuchElementException ex) {
+      PageObjectLogging.log("Emoticon " + emoticon + " on chat not displayed", ex, true);
+      return false;
+    }
+  }
+
+  public WebElement getMessage(String message) {
+    By by = By.xpath(String.format(PATH_MESSAGE_ON_CHAT, message));
+    return wait.forElementPresent(by);
+  }
+
+  private WebElement userPostedEmoticon(String emoticonName) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "userPostedEmoticon" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)